example/Router.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Equip\Queue\Exception\HandlerException;
+use Equip\Queue\Exception\RouterException;
+use Equip\Queue\RouterInterface;
+
+class Router implements RouterInterface
```

Comment:
Ah, didn't notice the path. Feel free to disregard. Though, is this implementation not usable on its own outside of being an example? Are we going to expect that library users will implement their own router classes?